### PR TITLE
Update local deploy:fork scripts

### PR DESCRIPTION
**Pull request by lazarmitic**

### CODE REVIEW

File: scripts/deposit.local.fork.ts [hunk 1]
```diff
@@ -53,13 +53,13 @@ class DepositLocalForked {
   }
 
   private async depositIntoPool(poolType: PoolType, ybt: ERC20, bt: ERC20, depositBacking: boolean, poolDepositConfig: DeployedPoolInfo) {
-    const principalShareToken = await ethers.getContractAt('PrincipalShare', poolDepositConfig.principalShare);
-    const yieldShareToken = await ethers.getContractAt('YieldShare', poolDepositConfig.yieldShare);
-    const poolSharePrincipal = new PoolShare('PoolShare', principalShareToken);
-    const poolShareYield = new PoolShare('PoolShare', yieldShareToken);
+    const principalShareToken = await ethers.getContractAt('PrincipalShare', poolDepositConfig.principalShareAddress);
+    const yieldShareToken = await ethers.getContractAt('YieldShare', poolDepositConfig.yieldShareAddress);
+    const poolSharePrincipal = new PoolShare('PoolShare', 18, principalShareToken);
```
Comment (RedFox20) [inline scripts/deposit.local.fork.ts:58]: 18 is not always valid, this should be `bt.decimals` for correctness sake 😃

File: scripts/deploy.local.fork.ts [hunk 2]
```diff
@@ -101,7 +142,7 @@ class DeployLocalForked {
     this.controller = await TempusController.deploy(this.owner);
     this.stats = await ContractBase.deployContract("Stats");
 
-    console.log('Deploying Aave Pool - aDAI - 1 year duration...');
+    /*console.log('Deploying Aave Pool - aDAI - 1 year duration...');
     await this.deployPool({
       poolType: PoolType.Aave,
       owner: this.owner,
```
[the remark this answers is not in the document]
Comment (lazarmitic) [reply scripts/deploy.local.fork.ts]: Removed commented out pool deployment code altogether.

File: scripts/deposit.local.fork.ts (hunk 1 above)
Comment (lazarmitic) [reply scripts/deposit.local.fork.ts]: Fixed, thanks for the feedback :D

**Comment by lazarmitic** [comment]
Rebase done

**Review by mijovic — APPROVED** [review]

**Pull request merged**